core/statistics.c

```diff
@@ -326,16 +326,18 @@ void get_selected_dives_text(char *buffer, size_t size)
 
 #define SOME_GAS 5000 // 5bar drop in cylinder pressure makes cylinder used
 
-bool is_cylinder_used(struct dive *dive, int idx)
+bool is_cylinder_used(struct dive *dive, int idx, bool ignore_plannned)
```

Comment:
That seems like a very invasive change for a special case. Couldn't this be better handled in the one call site where this matters?